content/en/hosting/analytics/setup-docker-compose.md

```diff
@@ -24,6 +24,8 @@ This guide will walk you through setting up a deployment of CHT Sync with the CH
 
 In the `cht-sync` folder, copy the values from the `env.template` file to a `.env` file. For more information, see the references on the [Environment variables page]({{< relref "hosting/analytics/environment-variables" >}}).
 
+For production CHT Core deployments, most likely the port will need to be set to `443` like this: `COUCHDB_PORT=443`. This is because CHT Core uses an `nginx` [reverse proxy]({{< relref "core/overview/architecture#overview" >}}) on port `443`, instead the default `5984` port used in a stand alone CouchDB instance which the `env.template` [has](https://github.com/medic/cht-sync/blob/bbe9801b678761165262d0f1325007fc9eae31cf/env.template#L22).
```

Comment:
```suggestion
For production CHT Core deployments, most likely, the port will need to be set to `443` like this: `COUCHDB_PORT=443`. This is because CHT Core uses an `nginx` [reverse proxy]({{< relref "core/overview/architecture#overview" >}}) on port `443`, instead of the default `5984` port used in a stand alone CouchDB instance which the `env.template` [has]({{< relref "hosting/analytics/environment-variables" >}}).
```